modules/api/src/main/java/org/eclipse/xpanse/api/config/DataModelConverter.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+package org.eclipse.xpanse.api.config;
+
+import java.util.Objects;
+import org.eclipse.xpanse.api.controllers.ServiceCatalogApi;
+import org.eclipse.xpanse.modules.database.servicetemplate.ServiceTemplateEntity;
+import org.eclipse.xpanse.modules.models.servicetemplate.view.ServiceTemplateDetailVo;
+import org.springframework.beans.BeanUtils;
+import org.springframework.hateoas.server.mvc.WebMvcLinkBuilder;
+
+/**
+ * DataModelConverter.
+ */
+public class DataModelConverter {
```

Comment:
we need a concrete name for this class otherwise this class might get too big with too many converters. 
We can call it -> `ServiceTemplateEntityConverter` -> then the class call have all converters related to `ServiceTemplateEntity`

Reply:
Updated.